### DISCUSSION TITLE
Moment.js

### DISCUSSION THREAD

**Discussion by jrpetersjr** [Lednerb/bilberry-hugo-theme#509] [General]
Hi,

What else besides localization is Moment.js used for in this project?

**Comment by igor-baiborodine** [comment]
If I'm correct, it's used **only** for date localization.

**Comment by jrpetersjr** [comment]
How do I test this? I don't think my testing is working.

**Reply by igor-baiborodine** [reply]
What exactly do you want to test? Please clarify.

**Comment by jrpetersjr** [comment]
I figured it out. Looked through the code found the moment code.
